content/assignments/techjobs-console/_index.md

```diff
@@ -0,0 +1,477 @@
+---
+title: "Assignment #1: Tech Jobs Console"
+date: 2023-01-03T10:42:56-06:00
+draft: false
+weight: 3
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Introduction
+
+Congratulations! Based on your hard work and strong coding skills, you have
+been brought on as an apprentice to the LaunchCode Tech Team. You will be
+paired with a mentor to help you get comfortable and continue learning.
+
+The Company Team at LaunchCode works with employer partners to match qualified
+programmers with apprenticeships. They asked for a new tool to be built to
+help them easily manage data for currently available jobs. Over the next few
+weeks, you will help them build this application alongside mentors from the
+Tech Team.
+
+This first project will be a simple proof-of-concept prototype. It won't be
```

Comment:
Proof of Concept - POC and Prototype represent two different phases of development. Might be nice too elaborate  a little to get students use to the vocabulary and principles in the development process.